rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -1606,6 +1604,8 @@ rmw_create_client(
     return nullptr;
   }
 
+  client_data->entity->copy_gid(client_data->client_gid);
+
```

Comment:
Technically can get rid of `client_data->client_gid` like we did in `PublisherData`?

Reply:
Good call; this is now removed in ff128e348d4acba6ab7b73ad7c7545cd0367818e